### ISSUE TITLE
[Request] Ender Watcher oreDict support

### ISSUE THREAD

**Issue by Gambigobilla** [talonos2/BlightfallTracker#167]
Currently Ender Watcher (spotloader) can only be crafted with gold nuggets, my suggestion is change 
minecraft:gold_nugget to ore:nuggetGold so we can use Thaumcraft gold coins which can be already used in other recipes that requires gold nuggets. Also saves the time converting coins to nugget (while at it can you add 9*coins == gold ingot recipe). Also with this recipe change we save 21 bytes of drive space.

Suggested recipe
mods.thaumcraft.Arcane.addShaped("CHICKENCHUNKS", ChickenChunks:chickenChunkLoader:1 \* 2, "ordo 16",
[[null, ore:nuggetGold,null],
[ore:nuggetGold, minecraft:ender_pearl, ore:nuggetGold],
[null, minecraft:obsidian, null]]);

**Comment by talonos2** [comment]
Thamcraft gold coins are weird. For example, you cannot make a gold ingot
out of 9 gold coins. But they work in other places as if they were gold
nuggets themselves. I tried to do what made sense and stay internally
consistent, but to be honest, I don't know what Azanor's intent was with
the gold coins. I might make this change, if only because the one player
who stated an opinion on the subject said they preferred it.

But protip: Gold coins have 1 lucrum essentia each in it. This means using
alchemical duplication, you can get three gold nuggets out of every gold
coin if you have a bunch of metallum essentia lying around. Unless you
already have infinite resources (through the bedrock dimension or mark of
the falling tower or something) this is a great way to stretch your gold.
(If you _do_ have infinite gold, then yeah, gold coins on chunk loaders is
a nice convenience thing)

**Comment by shade-of-ox** [comment]
Yeah, oreberries are weird as well. You can't use them to craft ingots, but you can use them in Thaumcraft recipes.

Aside from using Alchemy to convert them, there are a few things you can do:
Some villagers will trade emeralds for gold coins. This is... uh... not useful at all, really.
I believe there's a glitch where storage shelves where if you already have gold nuggets in there and you put in some gold coins they'll automagically get turned into gold nuggets. Saves on furnace fuel!
However, if you have a bunch of metallum, use alchemy instead.
